.github/workflows/release_workflow.yml

```
@@ -0,0 +1,59 @@
+# Give your test a name!
+name: ReleaseWorkflow
+
+ runs every two months on the 25th around 1300
+on:
```

```suggestion
on:
  workflow_dispatch:
```

I would also consider adding the workflow_dispatch event trigger here in case we want to manually create a release

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

---

I've added this now. Didn't know you could have multiple event triggers :)